robotpy_build/pyproject_configs.py

```diff
@@ -56,6 +60,16 @@ class MavenLibDownload(Model):
     #: Version of artifact to download
     version: str
 
+    #: Configure the sources classifier
+    sources_classifier: str = "sources"
+
+    #: When set, download sources instead of downloading libraries. When
+    #: using this, you need to manually add the sources to the configuration
+    #: to be compiled via :attr:`sources`.
+    use_sources: bool = False
+
+    # common with Download
```

Comment:
Can't we avoid this repetition by subclassing?

Reply:
I thought about it, but then the documentation ends up being less clear.